viewer/webpack.config.js

```diff
@@ -86,7 +86,7 @@ module.exports = env => {
       globalObject: `(typeof self !== 'undefined' ? self : this)`,
       libraryTarget: 'umd',
     },
-    devtool: development ? "inline-cheap-module-source-map" : "source-map",
+    devtool: development ? "inline-source-map" : "source-map",
```

Comment:
This doesn't really belong here, but was added because source maps didn't work in chrome.